### PR TITLE
v0.6.0 release notes

### PR DESCRIPTION
**Pull request by mangelajo**

### CODE REVIEW

File: src/content/releases/_index.en.md [hunk 1]
```diff
@@ -5,6 +5,22 @@ pre = "<b>5. </b>"
 weight = 25
 +++
 
+## v0.6.0 Lighthouse improvements
```
Comment (dfarrell07) [inline src/content/releases/_index.en.md:8]: MD linting is failing because you need a blank line after this heading.

**Comment by dfarrell07** [comment]
Do we want to mention something like "code quality improvements, added linters"?

File: src/content/releases/_index.en.md [hunk 2]
```diff
@@ -5,6 +5,22 @@ pre = "<b>5. </b>"
...
+> This release is mainly focused in support for headless for Services and StatefulSets in Lighthouse,
+> and A/P HA enhancements and general stability for Submariner.
+
+* **Lighthouse** has been enhanced to:
+  * Be aware of the local cluster Gateway connectivity, and now does not announce
+    IP addresses or disconnected remote clusters.
```
Comment (sridhargaddam) [inline src/content/releases/_index.en.md:14]:
```suggestion
  * Be aware of the local cluster Gateway connectivity, and now does not announce
    IP addresses for disconnected remote clusters.
```

File: src/content/releases/_index.en.md [hunk 3]
```diff
@@ -5,6 +5,22 @@ pre = "<b>5. </b>"
...
+  * Support Headless services
+  * Stateful sets
+* **subctl** auto-detects the cluster-id from the kubeconfig file information when possible.
+* HA improvements for *Submariner*, the shutdown path of all agents has been analized, and
+  fixed in some cases, improving the shutdown when an agent needs to be moved to a different
+  node.
```
Comment (sridhargaddam) [inline src/content/releases/_index.en.md:20]:
```suggestion
* HA improvements for *Submariner*. Graceful shutdown of Submariner Pods which will help in improving the downtime during Gateway failover. 
```

File: src/content/releases/_index.en.md [hunk 4]
```diff
@@ -5,6 +5,22 @@ pre = "<b>5. </b>"
...
+* Minimum **K8s** version is now 1.17, as we begin support for EndpointSlices in Lighthouse
+* Multiple submariner bugfixes since 0.5.0
+
```
Comment (sridhargaddam) [inline src/content/releases/_index.en.md:23]: * Hostnetwork to remoteService support fixes for AWS Cluster.

Notes: This is just for info. We fixed few edge cases where the hostnetwork to remoteService use-case was broken on AWS cluster. This is fixed via the following PRs.
Merged: https://github.com/submariner-io/submariner/pull/737
Merged: https://github.com/submariner-io/submariner/pull/738

File: src/content/releases/_index.en.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mangelajo) [reply src/content/releases/_index.en.md:8]: :+1: I missed that

File: src/content/releases/_index.en.md [hunk 5]
```diff
@@ -5,6 +5,20 @@ pre = "<b>5. </b>"
...
 
+## v0.6.0 Improved Submariner High Availability and various Lighthouse enhancements
+> This release mainly focused on support for headless Services and StatefulSets in Lighthouse,
+> as well as improving Submariner's High Availability (HA).
+
+* **Lighthouse** has been enhanced to:
+  * Be aware of the local cluster Gateway connectivity, and now does not announce
+    IP addresses for disconnected remote clusters.
+  * Support headless Services
+  * StatefulSets
```
Comment (vthapar) [inline src/content/releases/_index.en.md:16]:
```suggestion
  * Be aware of pods' availability for a service, and now doesn't announce IP addresses for services that have zero active pods
```
StatefulSets not yet, will likely come in 0.6.1

**Review by vthapar — CHANGES_REQUESTED** [review]

File: src/content/releases/_index.en.md (hunk 4 above)
Comment (mangelajo) [reply src/content/releases/_index.en.md:25]: Done.

File: src/content/releases/_index.en.md (hunk 5 above)
Comment (mangelajo) [reply src/content/releases/_index.en.md:17]: Why 0.6.1? we have released 0.6.0-rc0... we can wait to release 0.6.0, right?, lets talk on team meeting :)

**Comment by dfarrell07** [comment]
MD linting doing its job :muscle: :wink:

```
src/content/releases/_index.en.md: 19: MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
src/content/releases/_index.en.md: 23: MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
src/content/releases/_index.en.md: 24: MD034/no-bare-urls Bare URL used [Context: "https://github.com/submariner-..."]
```

File: src/content/releases/_index.en.md (hunk 5 above)
Comment (tpantelis) [reply src/content/releases/_index.en.md:17]: We decided on 0.7.0 for StatefulSets.

File: src/content/releases/_index.en.md [hunk 6]
```diff
@@ -1,10 +1,28 @@
...
-date = 2020-05-04T16:50:16+02:00
+date = 2020-08-24T11:35:16+02:00
 title = "Releases"
 pre = "<b>5. </b>"
 weight = 25
 +++
 
+## v0.6.0 Improved Submariner High Availability and various Lighthouse enhancements
+
+> This release mainly focused on support for headless Services and StatefulSets in Lighthouse,
```
Comment (tpantelis) [inline src/content/releases/_index.en.md:10]:
```suggestion
> This release mainly focused on support for headless Services in Lighthouse,
```

File: src/content/releases/_index.en.md [hunk 7]
```diff
@@ -1,10 +1,28 @@
...
+> as well as improving Submariner's High Availability (HA).
+
+* **Lighthouse** has been enhanced to:
+  * Be aware of the local cluster Gateway connectivity, and now does not announce
+    IP addresses for disconnected remote clusters.
+  * Support headless Services
+  * StatefulSets
+* **subctl** auto-detects the cluster-id from the kubeconfig file information when possible.
+* HA improvements for *Submariner*. Graceful shutdown of Submariner Pods which will help in improving the downtime during Gateway failover. 
+* Minimum **K8s** version is now 1.17, as we begin support for EndpointSlices in Lighthouse
```
Comment (tpantelis) [inline src/content/releases/_index.en.md:20]:
```suggestion
* Minimum **K8s** version is now 1.17, as we begin support for EndpointSlices in Lighthouse.
```

File: src/content/releases/_index.en.md [hunk 8]
```diff
@@ -1,10 +1,28 @@
...
+> This release mainly focused on support for headless Services and StatefulSets in Lighthouse,
+> as well as improving Submariner's High Availability (HA).
+
+* **Lighthouse** has been enhanced to:
+  * Be aware of the local cluster Gateway connectivity, and now does not announce
+    IP addresses for disconnected remote clusters.
+  * Support headless Services
+  * StatefulSets
+* **subctl** auto-detects the cluster-id from the kubeconfig file information when possible.
+* HA improvements for *Submariner*. Graceful shutdown of Submariner Pods which will help in improving the downtime during Gateway failover.
```
Comment (tpantelis) [inline src/content/releases/_index.en.md:19]:
```suggestion
* The Submariner Pods now shutdown gracefully and do proper cleanup which reduces the downtime during Gateway failover. 
```

File: src/content/releases/_index.en.md [hunk 9]
```diff
@@ -1,10 +1,28 @@
...
+
+> This release mainly focused on support for headless Services and StatefulSets in Lighthouse,
+> as well as improving Submariner's High Availability (HA).
+
+* **Lighthouse** has been enhanced to:
+  * Be aware of the local cluster Gateway connectivity, and now does not announce
+    IP addresses for disconnected remote clusters.
+  * Support headless Services
+  * StatefulSets
+* **subctl** auto-detects the cluster-id from the kubeconfig file information when possible.
```
Comment (tpantelis) [inline src/content/releases/_index.en.md:18]:
```suggestion
* **subctl** auto-detects the cluster ID from the kubeconfig file information when possible.
```

File: src/content/releases/_index.en.md [hunk 10]
```diff
@@ -1,10 +1,28 @@
...
+* HA improvements for *Submariner*. Graceful shutdown of Submariner Pods which will help in improving the downtime during Gateway failover. 
+* Minimum **K8s** version is now 1.17, as we begin support for EndpointSlices in Lighthouse
+* HostNetwork to remote Service support fixes for AWS Cluster. [Submariner Bug 736][Bug-736]
+
+
+[Bug-736][https://github.com/submariner-io/submariner/issues/736]
```
Comment (tpantelis) [inline src/content/releases/_index.en.md:24]:
```suggestion
* HostNetwork to remote Service connectivity fixes for AWS Cluster ([Issue 736](https://github.com/submariner-io/submariner/issues/736)).
```

File: src/content/releases/_index.en.md [hunk 11]
```diff
@@ -1,10 +1,26 @@
...
 pre = "<b>5. </b>"
 weight = 25
 +++
 
+## v0.6.0 Improved Submariner High Availability and various Lighthouse enhancements
+
+> This release mainly focused on support for headless Services in Lighthouse,
+> as well as improving Submariner's High Availability (HA).
+
+* **Lighthouse** has been enhanced to:
```
Comment (sridhargaddam) [inline src/content/releases/_index.en.md:13]: @mangelajo Are you planning to include that "Service Discovery" will be enabled by default while joining clusters?

Comment (mangelajo) [reply src/content/releases/_index.en.md:20]: L30 :-) may be I shall move it to this section instead?

Comment (mangelajo) [reply src/content/releases/_index.en.md:20]: Done.

File: src/content/releases/_index.en.md [hunk 12]
```diff
@@ -1,10 +1,35 @@
...
+
+{{% notice warning %}}
+  The DNS domains have been updated from %service.%namespace.svc.supercluster.local to
+  %service.%namespace.svc.clusterset.local according to U/S K8s MultiCluster SIG poll.
+  Both domains will be supported for 0.6.0, but 0.7.0 will remove support for supercluster.local,
+  please update your deployments and applications.
+{{% /notice %}}
+
+* **Lighthouse** has been enhanced to:
+  * Be aware of the local cluster Gateway connectivity, and now does not announce.
```
Comment (vthapar) [inline src/content/releases/_index.en.md:21]:
```suggestion
  * Be aware of the local cluster Gateway connectivity, and now does not announce
```

File: src/content/releases/_index.en.md [hunk 13]
```diff
@@ -1,10 +1,35 @@
...
+    IP addresses for disconnected remote clusters.
+  * Support headless Services-
```
Comment (vthapar) [inline src/content/releases/_index.en.md:23]:
```suggestion
  * Support Headless Services
```

File: src/content/releases/_index.en.md [hunk 14]
```diff
@@ -1,10 +1,35 @@
...
+  * Be aware of pods' availability for a service, and now doesn't announce IP addresses for services that have zero active pods.
+  * Round Robin IP resolution for services available in multiple clusters.
```
Comment (vthapar) [inline src/content/releases/_index.en.md:25]:
```suggestion
  * Use Round Robin IP resolution for services available in multiple clusters.
```

File: src/content/releases/_index.en.md [hunk 15]
```diff
@@ -1,10 +1,35 @@
...
+  * Service discovery is enabled by default for **subctl** deployments.
```
Comment (vthapar) [inline src/content/releases/_index.en.md:26]:
```suggestion
  * Enable Service discovery by default for **subctl** deployments.
```

**Review by vthapar — CHANGES_REQUESTED** [review]

File: src/content/releases/_index.en.md (hunk 12 above)
Comment (mangelajo) [reply src/content/releases/_index.en.md:21]: ooppsie

**Review by nyechiel — COMMENTED** [review]
We still missing a point about Prometheus metrics

File: src/content/releases/_index.en.md [hunk 16]
```diff
@@ -1,10 +1,35 @@
...
 +++
 
+## v0.6.0 Improved Submariner High Availability and various Lighthouse enhancements
+
+> This release mainly focused on support for headless Services in Lighthouse,
+> as well as improving Submariner's High Availability (HA).
+
+{{% notice warning %}}
+  The DNS domains have been updated from %service.%namespace.svc.supercluster.local to
+  %service.%namespace.svc.clusterset.local according to U/S K8s MultiCluster SIG poll.
```
[the remark this answers is not in the document]
Comment (tpantelis) [reply src/content/releases/_index.en.md:15]:
```suggestion
  %service.%namespace.svc.clusterset.local to align with the Kubernetes MultiCluster SIG.
```

File: src/content/releases/_index.en.md [hunk 17]
```diff
@@ -1,10 +1,35 @@
...
+  Both domains will be supported for 0.6.0, but 0.7.0 will remove support for supercluster.local,
+  please update your deployments and applications.
```
Comment (tpantelis) [inline src/content/releases/_index.en.md:17]:
```suggestion
  Please update your deployments and applications.
```

File: src/content/releases/_index.en.md [hunk 18]
```diff
@@ -1,10 +1,34 @@
...
 +++
 
+## v0.6.0 Improved Submariner High Availability and various Lighthouse enhancements
+
+> This release mainly focused on support for headless Services in Lighthouse,
+> as well as improving Submariner's High Availability (HA).
+
+{{% notice warning %}}
+  The DNS domains have been updated from \<service\>.\<namespace\>.svc.**supercluster**.local to
+  \<service\>.\<namespace\>.svc.**clusterset**.local according to align with the Kubernetes MultiCluster SIG poll.
```
Comment (skitt) [inline src/content/releases/_index.en.md:15]:
```suggestion
  \<service\>.\<namespace\>.svc.**clusterset**.local to align with the change in Kubernetes MultiCluster Service API.
```

The poll ultimately doesn’t matter, the API does ;-).

File: src/content/releases/_index.en.md [hunk 19]
```diff
@@ -1,10 +1,34 @@
...
+
+* **Lighthouse** has been enhanced to:
+  * Be aware of the local cluster Gateway connectivity so as not to announce the IP addresses for disconnected remote clusters.
+  * Support headless Services
+  * Be aware of a Service's backend Pods so as not to announce IP addresses for Services that have no active pods.
+  * Use Round Robin IP resolution for services available in multiple clusters.
+  * Enable Service discovery by default for **subctl** deployments.
+* **subctl** auto-detects the cluster ID from the kubeconfig file information when possible.
+* The Submariner Pods now shutdown gracefully and do proper cleanup which reduces the downtime during Gateway failover.
+* Minimum Kubernetes version is now 1.17.
```
Comment (skitt) [inline src/content/releases/_index.en.md:28]:
```suggestion
* The operator now automatically exports Prometheus metrics; these integrate seamlessly with OpenShift Prometheus if user workload monitoring is enabled, and can be included in any other Prometheus setup.
* Minimum Kubernetes version is now 1.17.
```

**Review by skitt — APPROVED** [review]

File: src/content/releases/_index.en.md [hunk 20]
```diff
@@ -1,10 +1,34 @@
...
+{{% notice warning %}}
+  The DNS domains have been updated from \<service\>.\<namespace\>.svc.**supercluster**.local to
+  \<service\>.\<namespace\>.svc.**clusterset**.local according to align with the Kubernetes MultiCluster SIG poll.
+  Both domains will be supported for 0.6.0 but 0.7.0 will remove support for **supercluster**.local.
+  Please update your deployments and applications.
+{{% /notice %}}
+
+* **Lighthouse** has been enhanced to:
+  * Be aware of the local cluster Gateway connectivity so as not to announce the IP addresses for disconnected remote clusters.
+  * Support headless Services
```
Comment (vthapar) [inline src/content/releases/_index.en.md:22]:
```suggestion
  * Support headless Services for non-gloablnet deployments. Currently globalnet + headless services is not supported.
```

File: src/content/releases/_index.en.md [hunk 21]
```diff
@@ -1,10 +1,34 @@
...
+  * Be aware of a Service's backend Pods so as not to announce IP addresses for Services that have no active pods.
+  * Use Round Robin IP resolution for services available in multiple clusters.
+  * Enable Service discovery by default for **subctl** deployments.
+* **subctl** auto-detects the cluster ID from the kubeconfig file information when possible.
+* The Submariner Pods now shutdown gracefully and do proper cleanup which reduces the downtime during Gateway failover.
+* Minimum Kubernetes version is now 1.17.
+* HostNetwork to remote Service connectivity fixes for AWS Cluster ([Issue 736](https://github.com/submariner-io/submariner/issues/736)).
```
Comment (dfarrell07) [inline src/content/releases/_index.en.md:29]:
```suggestion
* HostNetwork to remote Service connectivity fixes for AWS clusters ([Issue 736](https://github.com/submariner-io/submariner/issues/736)).
```

Comment (mangelajo) [inline src/content/releases/_index.en.md]:
```suggestion
* HostNetwork to remote Service connectivity fixes for AWS clusters ([Issue 736](https://github.com/submariner-io/submariner/issues/736)).
```

File: src/content/releases/_index.en.md [hunk 22]
```diff
@@ -1,10 +1,35 @@
...
+{{% notice warning %}}
+  The DNS domains have been updated from \<service\>.\<namespace\>.svc.**supercluster**.local to
+  \<service\>.\<namespace\>.svc.**clusterset**.local to align with the change in Kubernetes MultiCluster Service API.
+  Both domains will be supported for 0.6.0 but 0.7.0 will remove support for **supercluster**.local.
+  Please update your deployments and applications.
+{{% /notice %}}
+
+* **Lighthouse** has been enhanced to:
+  * Be aware of the local cluster Gateway connectivity so as not to announce the IP addresses for disconnected remote clusters.
+  * Support headless Services for non-gloablnet deployments. Currently globalnet + headless services is not supported.
```
Comment (tpantelis) [inline src/content/releases/_index.en.md:22]:
```suggestion
  * Support headless Services for non-**Globalnet** deployments. Support for **Globalnet** will be available in a future release.
```

**Review by mkolesnik — APPROVED** [review]

**Review by sridhargaddam — APPROVED** [review]

**Review by nyechiel — APPROVED** [review]

**Pull request merged**